tools/install.sh

```diff
@@ -84,7 +84,9 @@ else
     echo ""
     echo "PHP Composer does not appear to be installed. Please install it before running this script."
     echo ""
-    echo "(e.g. curl -sS https://getcomposer.org/installer | php)"
+    echo "(e.g. wget https://getcomposer.org/installer"
+    echo "php installer --install-dir=/usr/local/bin --filename=composer)"
```

Comment:
Is there a reason this can't still use the pipe? You're only going to install it once, I don't see much reason to save thescript.

Reply:
not really, just have to be sure the pass the argument to install as installing with default value result to composer being installed in the current directory with another filename. Will work on that.